specs/webSocket.spec.ts

```diff
@@ -259,14 +262,11 @@ describe("webSocket Manager tests", () => {
         comapiConfig.webSocketBase = "ws://junk";
 
         webSocketManager.connect()
-            .then(function () {
-                fail("Should not be in here");
-                done();
-            })
-            .catch(function () {
+            .then(function (connected) {
                 let isConnected = webSocketManager.isConnected();
                 console.log("isConnected = " + isConnected);
```

Comment:
Logging - `ILogger`?

Reply:
Hmm, not sure i am bothered about spec files logging ...

Reply:
😂 yeah ignore that one!